core/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -135,7 +135,10 @@ private static BulkItemResultHolder executeIndexRequest(final IndexRequest index
         if (indexResult.hasFailure()) {
             return new BulkItemResultHolder(null, indexResult, bulkItemRequest);
         } else {
-            IndexResponse response = new IndexResponse(primary.shardId(), indexRequest.type(), indexRequest.id(),
+            String requestReturn = indexRequest.source().utf8ToString();
+            requestReturn = requestReturn.replace("\n", "");
+            requestReturn = requestReturn.replace("\"", "");
```

Comment:
Also, `requestReturn` is  a confusing variable name for me. Maybe just `request` or `requestToReturn`?